week4/src/pages/components/displaypokemon.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Pokemon } from '../schema/pokemonSchema'
+
+interface pokemonProps {
+  pokemon: Pokemon
+}
+
+const DisplayPokemon: React.FC<pokemonProps> = ({ pokemon }) => {
+
+  return (
+    <div className="pokemonItem">
+      <img src={pokemon.image} alt={pokemon.name}></img>
+      <div style={{ textTransform: 'uppercase' }}>{pokemon.name}</div>
+      <br></br>
+      <div className="types">
+        {pokemon.type.map((type: string, index: number) => (
+          <div key={index}>
+            <svg width={30} height={30} aria-hidden="true">
+            <use xlinkHref={`sprite.svg#${type}`} />
```

Comment:
xlinkHref is depreciated https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href
Can you look for some other alternatives?

Reply:
sure